portfolio/src/main/webapp/script.js

```diff
@@ -32,19 +32,32 @@ function addRandomQuote() {
  * Add all comments to the page
  */
 function addComments() {
```

Comment:
nit: This doesn't really 'add' comments, but displays them. Consider renaming it to `fetchComments` or `getComments`.